Workshop.md

```diff
@@ -0,0 +1,477 @@
+
+### 1. Run the ToDo-Backend Tests:
+
+1. Open the tests in a web browser: `open ~/todo-backend-js-spec/index.html`
+2. Set a "test target root" of `http://localhost:8080/todos`
+3. Click "run tests".
+
+All the tests should fail. The first error reported should be as follows:
+
+:x: `the api root responds to a GET (i.e. the server is up and accessible, CORS headers are set up)`
+
+```
+AssertionError: expected promise to be fulfilled but it was rejected with [Error: 
+
+GET http://localhost:8080/todos
+FAILED
+
+The browser failed entirely when make an AJAX request.
+```
+
+This shows that the tests made a `GET` request to `http://localhost.com:8080/todos`, but it failed with no response. This is expected as there is no server yet.
+
+In the instructions below, reloading the page will allow you to re-run the ToDo-Backend tests.
+
+## Building a Kitura server
+
+Implementing a compliant ToDo Backend is an incremental task, with the aim being to pass more of the testsuite at each step. The first step is to build and run your Kitura server to response to requests.
+
+### 1. Run your Kitura server
+
+1. Open the ToDoServer project in Xcode. 
+    ```
+    cd ~/ToDoBackend/ToDoServer  
+    open ToDoServer.xcodeproj
+    ```
+
+2. Run your Kitura server in Xcode:
+    1) Change the selected target from "ToDoServer-Package" to the "TodoServer" executable. 2) Press the `Run` button or use the `⌘+R` key shortcut. 3) Select "Allow incoming network connections" if you are prompted.
+
+3. Check that some of the standard Kitura URLs are running:
+    * Kitura splash screen: [http://localhost:8080/](http://localhost:8080/)
+    * Kitura monitoring dashboard: [http://localhost:8080/swiftmetrics-dash/](http://localhost:8080/swiftmetrics-dash/)
+    * Kitura health API: [http://localhost:8080/health](http://localhost:8080/health)
+
+### 2. Add support for OpenAPI
+
+[OpenAPI](https://www.openapis.org/) is the most popular way to document RESTful web services. The OpenAPI ecosystem provides a broad range of tools and services for developers across the API lifecycle.
+
+Kitura provides a package which makes it easy to add OpenAPI support to your application. Let's add OpenAPI.
+
+1. Open the `ToDoServer` > `Package.swift` file
+2. Add the following to the end of the dependencies section of the Package.swift file:
+    ```swift
+        .package(url: "https://github.com/IBM-Swift/Kitura-OpenAPI.git", from: "1.0.0"),
+    ```
+3. Update the target dependencies for the "Application" target to the following (note the lack of hyphen in KituraOpenAPI):
+    ```swift
+    .target(name: "Application", dependencies: ["KituraOpenAPI", "Kitura", "Configuration", "CloudEnvironment", "Health", "SwiftMetrics" ]),
```

Comment:
I'm not sure whether this varies by the method you choose to follow in the README.md but I don't have "Configuration" as a target dependency in my `Package.swift` file which was generated by `kitura init`.